src/inner-slider.jsx

```diff
@@ -19,11 +19,25 @@ export var InnerSlider = React.createClass({
   getDefaultProps: function () {
     return defaultProps;
   },
-  componentDidMount: function () {
-    // Hack for autoplay -- Inspect Later
+  componentWillMount: function () {
```

Comment:
Here the `setState` doesn't change anything so a rerender of VDOM is not required.